Added EAK's Cookie Bear program (EAK;TS BEAR).

eswenson1 opened the pull request:
This version actually works.

larsbrinkhoff commented on build/misc.tcl:
```diff
@@ -695,6 +695,10 @@ respond "SUBJECT:" "COOKIE\r"
 expect ":KILL"
 
+# Cookie Bear (this one actually works)
+respond "*" "midas eak;ts bear_eak;bear\r"
```
This is missing a colon.

larsbrinkhoff reviewed (changes requested):
Please add file timestamps.

larsbrinkhoff commented:
Suggest adding to the commit comment some description of how to interact with the program.

eswenson1 commented:
I've added timestamps and updated the commit log with information on how to invoke BEAR. I also fixed the missing ":" in the TCL script.  @larsbrinkhoff Please let me know when you approve and I can merge.

eswenson1 commented:
@larsbrinkhoff This PR still says that there are requested changes that aren't yet resolved.  How do I indicate that I resolved all the issues you brought up?  Or is this something you have to do?

larsbrinkhoff reviewed (approved):
Looks nice now!

larsbrinkhoff commented:
I think they are resolved by someone clicking approve.

larsbrinkhoff commented:
All good here.
